src/cli/update_checker.rs

```diff
@@ -0,0 +1,59 @@
+use colored::Colorize;
+use update_informer::{registry, Check};
+use which::which;
+
+enum InstallationMethod {
+  Npm,
+  Brew,
+  Direct,
+}
+
+fn get_installation_method() -> InstallationMethod {
+  let output = std::process::Command::new("npm").arg("ls").arg("--global").output();
```

Comment:
use could actually install it locally to project and use it with `npx tailcall`.

I think we could validate some available env variables that set by npm and other package manager to figure out how program is ran.